.circleci/config.yml

```diff
@@ -0,0 +1,158 @@
+version: 2.1
+
+executors:
+
+  "vm-ubuntu-16-04":
+    machine:
+      image: "ubuntu-1604:201903-01"
+    environment:
+      build_aux_expected_GOHOSTOS: "linux"
+      build_aux_expected_GOHOSTARCH: "amd64"
+      build_aux_expected_FLOCK: "/usr/bin/flock"
+
+  "vm-macos-10-13":
+    macos:
+      xcode: "10.1.0"
+    environment:
+      build_aux_expected_GOHOSTOS: "darwin"
+      build_aux_expected_GOHOSTARCH: "amd64"
+      build_aux_expected_FLOCK: ""
+
+commands:
+
+  "apt-install":
+    parameters:
+      "packages":
+        type: string
+    steps:
+      - run:
+          command: |
+            sudo sed -i 's,//archive\.ubuntu\.com,//us-east-1.ec2.archive.ubuntu.com,g' /etc/apt/sources.list &&
+            sudo apt update &&
+            sudo apt install << parameters.packages >>
+
+  "brew-install":
+    parameters:
+      "packages":
+        type: string
+    steps:
+      - restore_cache:
+          name: "Restore brew cache"
+          keys:
+            - brew-{{ arch }}
+      - run: brew install << parameters.packages >>
+      - save_cache:
+          name: "Save brew cache"
+          key: brew-{{ arch }}
+          paths:
+            - "/usr/local/Homebrew"
+
+  "install-go":
+    parameters:
+      "version":
+        type: string
+        default: "1.13"
+    steps:
+      - run:
+          name: "Install Go"
+          command: |
+            curl https://dl.google.com/go/go1.12.$(uname -s | tr A-Z a-z)-amd64.tar.gz -o /tmp/go.tar.gz
```

Comment:
This should use the version parameter, right?

Reply:
Yes. Yes it should.